internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/controller/TestParentControllerWithMultiDataCenter.java

```diff
@@ -308,6 +308,71 @@ public void testUpdateStoreOnParentDoNotChangeIrrelevantConfig() {
     }
   }
 
+  @Test(timeOut = TEST_TIMEOUT)
+  public void testUpdateStoreOnParentEnableWriteCompute() {
+    String clusterName = CLUSTER_NAMES[0];
+    String storeName = Utils.getUniqueString("store");
+    String valueSchema = "{\n" + "  \"type\": \"record\",\n" + "  \"name\": \"TestRecord\",\n" + "  \"fields\": [\n"
+        + "    {\"name\": \"int_field\", \"type\": \"int\", \"default\": 0},\n"
```

Comment:
nit: I find this hardcoded schema everywhere in the integration test, would be nice to load and parse real avsc instead. Maybe I can have a cleanup PR to get rid of all these. Some are just duplicated.

Reply:
I think the problem ends up being that each module has it's own resources and will have a copy of the schemas. But that's still fewer duplicates than what we currently do.

We can improve that too by keeping schemas in one place and using symlinks in other modules